### DISCUSSION TITLE
The problem of using tauri in centos

### DISCUSSION THREAD

**Discussion by hanxiaohuivc** [tauri-apps/tauri#2498] [Q&A] [answered]
Hi everyone. I deployed tauri in centos7 and reported the following problems when using rust commands:

SyntaxError: Unexpected token'{'. import call expects exactly one argument.

My js code is as follows, these codes can run normally under windows:

<script>
     import {invoke} from'@tauri-apps/api/tauri'
     const TAURI=window.__TAURI__;
     const invoke = window.__TAURI__.invoke
     function jswebapi() {
         $.ajax({
             url: "https://jsonplaceholder.typicode.com/posts/1",
             type: "get",
             success: function (data) {
                 console.log(data);
                 let textId = $("#jswebapitext");
                 textId.val(JSON.stringify(data));
             }
         });
     }
</script>

How to modify in centos? Thank you.

**Comment by hanxiaohuivc** [comment, marked as answer]
I found the reason by myself. In Linux, you should use ‘const invoke = window.__TAURI_INVOKE__’ instead of ‘const invoke = window.__TAURI__.invoke’.

**Reply by amrbashir** [reply]
That's not right, `window.TAURI_INVOKE` is meant for internal usage, `window.__TAURI__.tauri.invoke` should work only when you have `withGlobalTauri` enabled

**Reply by FabianLars** [reply]
I think the problem boils down to the ES2021 target again (#2473), because centos 7 doesn't provide webkit2, only version 1 which probably doesn't support newer standards :/
